scopes/pkg/pkg/pkg.docs.md

```diff
@@ -1,35 +1,38 @@
 ---
-description: Bit Component package automation, packing and publishing.
+description: Generates, packs and publishes component packages
 labels: ['packages', 'aspect', 'pkg']
 ---
 
-Pkg aspect automates the creation, packing and publishing of packages of Bit components.
+Bit components can be thought of as a super-set of standard packaged node modules. Each component contains a consumable package in addition to its documentation, history and other information that enables it to be maintained independently.
 
-1. allows users to change (add) properties in the component's package.json
-2. allows other extensions to add new properties to the component's package.json
-3. allows an env to add new properxties to the component's package.json
-4. exposes a pack command to pack a component into a tar suitable for an npm registry
-5. exposes a publish command to publish components to a private registry
-6. utilizes PostTag hook to auto publish components after tag
+Components are not only consumed as standard packages but can also be [published to NPM](/docs/packages/publish-to-npm) or any other package registry (in addition to being 'exported' to a remote scope on a bit server).
 
-## Usage
+The 'pkg' aspect generates, packs and publishes component packages. That includes:
 
-### Configuration
+1. Allowing users to modify properties in the component's `package.json`
+2. Allowing other Bit extensions to add new properties to the component's `package.json`.
+3. Allowing a Bit environment to add new properties to the component's `package.json`.
+4. Exposing a `pack` command to pack a component into a tar suitable for an npm registry
+5. Exposing a `publish` command to publish components to a private registry
+6. Utilizing the `PostTag` hook to auto publish components after tag
```

Comment:
@giteden 
that's not accurate.
It's utilizing the builder register build task and register deploy tasks
```js
const dryRunTask = new PublishDryRunTask(PkgAspect.id, publisher, packer, logPublisher);
    const preparePackagesTask = new PreparePackagesTask(PkgAspect.id, logPublisher);
    const publishTask = new PublishTask(PkgAspect.id, publisher, packer, logPublisher);
    dryRunTask.dependencies = [BuildTaskHelper.serializeId(preparePackagesTask)];
    builder.registerBuildTasks([preparePackagesTask, dryRunTask]);
    builder.registerDeployTasks([publishTask]);
```